src/components/FlowRunMenu.vue

```diff
@@ -1,6 +1,10 @@
 <template>
   <p-icon-button-menu>
     <template #default>
+      <router-link :to="routes.customFlowRunCreate(flowRunId)">
+        <p-overflow-menu-item label="Custom run" />
```

Comment:
I'd call this "Copy run" or "Duplicate run"

Reply:
Or maybe "Run Again"?

Reply:
Both suggestions make sense. I'm trying to think if there are one or two words that can describe running the same thing with different parameters

Reply:
ChatGPT suggested 'Parametric execution/run' 😄

Reply:
Can you ask it to try again but give something simpler?

Reply:
It's not giving very good ideas.. It also suggested `Adjusted run`

Reply:
I vote we go with "Run Again"

Reply:
Run Again and Retry next to each other might be confusing together. I chose Custom Run because it matches what deployment does, and technically it takes you to the same page as customDeploymentRun, but with pre-filled params

<img width="215" alt="Screenshot 2023-02-03 at 10 45 48 AM" src="https://user-images.githubusercontent.com/40467112/216646134-2cc10941-2d40-4cf2-a35d-ef1d6d5d135a.png">

Reply:
Yeah I get that.  But custom run doesn't explain what it's doing.  How about Duplicate run?

Naming is hard and I think any of these are fine so I'll let you make the final decision here and we can go with it.

Reply:
Its basically the same as "Quick Run" or "Custom Run" on the deployments page. What if we copied the same ui? Made the Retry button have a dropdown same as the run button on a deployment.

<img width="240" alt="image" src="https://user-images.githubusercontent.com/6200442/216647016-dcc7e441-1d3e-4f1e-ab54-c5fce9503935.png">

Reply:
I think we'd have other naming questions then... I vote we keep it simple for now.

Reply:
But again, Marichka can have decision power here.